indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/supervisor/SeekableStreamSupervisor.java

```diff
@@ -3543,6 +3682,24 @@ protected void emitLag()
     }
   }
 
+
+  /**
+   * This method compute maxLag, totalLag and avgLag then fill in 'lags'
```

Comment:
Not sure what you mean by `fill in 'lags'`
```suggestion
   * This method computes maxLag, totalLag and avgLag 
```

Reply:
Thanks && Changed.